include/nbl/video/IAPIConnection.h

```diff
@@ -61,15 +60,38 @@ class NBL_API2 IAPIConnection : public core::IReferenceCounted
 
         const SFeatures& getEnabledFeatures() const { return m_enabledFeatures; }
 
-        const bool isRunningInRenderdoc() const { return m_rdoc_api; }
+        enum SDebuggerType
+        {
+            EDT_NONE,
+            EDT_RENDERDOC,
+            EDT_NGFX
```

Comment:
1. don't use the `S` prefix for enums, either no prefix or `E`
2. Use `enum class` and always remember to explicitly size the enum (so `: uint8_t`)
3. if you use enum class, then you don't need `EDT_` prefix (because class enum creates a scope for the enum)